src/bip39.rs

```diff
@@ -236,6 +236,10 @@ impl Bip39Secret {
             .collect::<Vec<_>>()
             .join(" ")
     }
+    /// Generate the seed phrase (mnemonic) from the secret.
+    pub fn to_seed_phrase(&self, dictionary: &Bip39Dictionary) -> String {
+        self.to_mnemonic(dictionary)
+    }
```

Comment:
Why do we need this function? That is, why not call `to_mnemonic` directly? Eg., I see you do this here https://github.com/asonnino/shamir-bip39/blob/64283e5669c5d7ea6923f2db9994ce9b4acc2271/src/main.rs#L240

Reply:
Good point. I was playing around with different things. I think this was leftover